### ISSUE TITLE
Undocumented `alwaysByViewport` option

### ISSUE THREAD

**Issue by GeoMarkou** [react-component/tooltip#363]
In the `align.overflow` configuration there is an `alwaysByViewport` option offered by [dom-align](https://github.com/yiminghe/dom-align):

> if alwaysByViewport is true, the it will adjust if node is not inside viewport

This solved an issue I was having with a large page that scrolled horizontally and vertically, the tooltip would disappear off the side of the page. It would be great if this was documented somewhere.

**Comment by karpo518** [comment]
When the trigger button is located near the border of the browser window, then the popup falls outside the window and creates a scroll. I was unable to resolve this issue using the option adalwaysByViewport, adjustX or adjustY

**Comment by samanthaamodeo-cognition** [comment]
I agree with wanting some documentation on this option. I had tried passing it as an option to align.overflow to solve some cut-off Tooltips. While it did seem to shift the tooltip closer to being fully in view, still did not fully resolve cut-off tooltips and/or scrollbars.
